dis_snek/client.py

```diff
@@ -1392,6 +1393,21 @@ async def get_member(self, user_id: "Snowflake_Type", guild_id: "Snowflake_Type"
         """
         return await self.cache.get_member(guild_id, user_id)
 
+    async def get_scheduled_event(
+        self, guild_id: "Snowflake_Type", scheduled_event_id: "Snowflake_Type", with_user_count: bool = False
+    ) -> "ScheduledEvent":
```

Comment:
This doesn't actually *need* to be in quotations, you *need* to use quoations when your type hint is imported through `TYPE_CHECKING` or if youre forward referencing 
(don't act on this, just mark as resolved)

Reply:
Thank's for the info.